internal/core/src/segcore/plan_c.cpp

```diff
@@ -36,7 +36,7 @@ CreateSearchPlanByExpr(CCollection c_col,
         auto status = CStatus();
         status.error_code = milvus::Success;
         status.error_msg = "";
-        auto plan = (CSearchPlan)res.release();
+        auto plan = static_cast<CSearchPlan>(res.release());
```

Comment:
hmm, just `CSearchPlan` and not `CSearchPlan*` ?

Reply:
That will throw error since CSearchPlan is defined as void*.